SpanishFilter/sections/specific.txt

```diff
@@ -198,6 +198,9 @@ epocanegocios.globo.com##div[class^="container-adv"]
 epocanegocios.globo.com##.adv-item
 epocanegocios.globo.com###pub-banner-audima
 epocanegocios.globo.com###pub-shopping
+acidadeon.com##div[class^="slb"]
```

Comment:
`acidadeon.com##.slb0` is enough because you already have `acidadeon.com##.ads`.